sharepoint/sharepoint-ps/sharepoint-online/Start-SPOCopilotAgentInsightsReport.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spocopilotagentinsightsreport
+applicable: SharePoint Online
+title: Start-SPOCopilotAgentInsightsReport
+schema: 2.0.0
+author: bhagatshweta
+ms.author: bhagatshweta
+ms.reviewer:
+manager: hikakar
+---
+ 
+# Start-SPOCopilotAgentInsightsReport
+
+## SYNOPSIS
+
+This cmdlet enables administrator to trigger the build of a new Copilot Agent Insight report for the last N days.
```

Comment:
>Agent Insight [](http://example.com/codeflow?start=72&length=13)

what's the right casing?

some places down below say "agent insight" (all lowercase) -- it should be consistent everywhere

Reply:
fixed the casing: using "agent insight" at all places.